skills/stash-cli/SKILL.md

```
@@ -457,13 +464,13 @@ Chunked, resumable, idempotent. Walks the table in keyset-pagination order, encr
 stash encrypt cutover --table users --column email
 ```
 
-**Preconditions:** the column is in the `backfilled` phase, **and** a pending EQL configuration exists.
+**EQL v2 only** — v3 has no cut-over: the application switches to `<col>_encrypted` by name, and running this command on a v3 column reports "not applicable" (exit 0) with the next step. For v2, the preconditions are: the column is in the `backfilled` phase, **and** a pending EQL configuration exists.
 
 In one transaction it renames `<col>` → `<col>_plaintext` and `<col>_encrypted` → `<col>`, advances the pending config to `encrypting`, activates it, and appends a `cut_over` event. With a Proxy URL configured (`--proxy-url` or `CIPHERSTASH_PROXY_URL`) it then calls `eql_v2.reload_config()` so Proxy picks up the new shape.
 
 > **After cutover, `<col>` holds ciphertext — the read path is not automatic.** Wire reads through the encryption client (`decryptModel(row, table)` for Drizzle, the `encryptedSupabaseV3` wrapper for Supabase — `encryptedSupabase` on the legacy v2 surface, otherwise `decrypt` / `bulkDecryptModels`) before returning values to callers. Skip this and your read paths hand raw EQL payloads to end users. The integration skill has the exact API. **CipherStash Proxy is the one exception** — it decrypts on the wire, so Proxy users need no application change. The cutover plan written by `stash plan` includes this read-path switch as an explicit step.
 
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)